app/services/create_proposed_organisation_edit.rb

```diff
@@ -34,8 +35,8 @@ def merge_in_non_published_fields
     end
   end
 
-  def send_email_to_superadmin_about_org_edit
-    superadmin_emails = user_klass.superadmins.pluck(:email)
-    mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails).deliver_now
-  end
+  # def send_email_to_superadmin_about_org_edit
+  #   superadmin_emails = user_klass.superadmins.pluck(:email)
+  #   mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails).deliver_now
+  # end
```

Comment:
rather than commenting this function out might we prefer the following:

```rb
def send_email_to_superadmin_about_org_edit
  superadmin_emails = user_klass.superadmin_emails
  mailing = mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails)
  mailing.deliver_now
end
```